middleware/cors.js

```diff
@@ -13,6 +13,19 @@ var cors = require('cors');
 
 exports.init = function(app, config, logger, callback) {
     var cfg = config.get('cors');
+    if (typeof cfg.origin === 'object') { //this is meant to be regex
```

Comment:
wow, stupid mistake. I didn't catch this because we are not using an array in our project